src/components/PDFView/PDFInfoMessage.tsx

```diff
@@ -1,25 +1,24 @@
-import PropTypes from 'prop-types';
 import React from 'react';
+import type {GestureResponderEvent} from 'react-native';
 import {View} from 'react-native';
 import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
 import Text from '@components/Text';
 import TextLink from '@components/TextLink';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import variables from '@styles/variables';
 
-const propTypes = {
+type PDFInfoMessageProps = {
     /** Callback function to indicate that PDF password form should be shown */
-    onShowForm: PropTypes.func.isRequired,
-
-    ...withLocalizePropTypes,
+    onShowForm: () => void | ((event: GestureResponderEvent) => void);
```

Comment:
```suggestion
    onShowForm: (event: GestureResponderEvent | KeyboardEvent) => void;
```

Import `KeyboardEvent` from `react`